doc/releases.md

```diff
@@ -127,64 +134,67 @@ YYYY-MM-DD io.js vx.y.z Release
 
 Notable changes:
 
-* Copy the notable changes list here
+* Copy the notable changes list here, reformatted for plain-text
 ```
 
-### 7. Tag and Sign the Release Commit
+### 6. Push to GitHub
 
-Tag the release as <b><code>vx.y.z</code></b> and sign **using the same GPG key that will be used to sign SHASUMS256.txt**.
+Note that it is not essential that the release builds be created from the io.js repository, they may be created from your own fork if you desire. It's preferable, but not essential that the commits remain the same between that used to build and the tagged commit in the io.js repository.
 
-```
-git tag -sm 'YYYY-MM-DD io.js vz.y.x Release' vx.y.z
-```
+### 7. Produce Release Builds
 
-### 8. Set Up For the Next Release
+Use **[iojs+release](https://jenkins-iojs.nodesource.com/job/iojs+release/)** to produce release artifacts. Enter the commit that you want to build from and select "release" for "disttype".
 
-Edit _src/node_version.h_ again and:
+Artefacts from each slave are uploaded to Jenkins and are available if further testing is required. Use this opportunity particularly to test OS X and Windows installers if there are any concerns. Click through to the individual slaves for a run to find the artifacts.
 
-* Increment `NODE_PATCH_VERSION` by one
-* Change `NODE_VERSION_IS_RELEASE` back to `0`
+All release slaves should achieve "SUCCESS" (and be blue, not red). A release with failures should not be promoted, there are likely problems to be investigated.
 
-Commit this change with:
+You can rebuild the release as many times as you need prior to promoting them if you encounter problems.
 
-```
-git commit -am 'Working on vx.y.z' # where 'z' is the incremented patch number
-```
+Note that you do not have to wait for the ARM builds if they are take longer than the others. It is only necessary to have the main Linux (x64 and x86), OS X .pkg and .tar.gz, Windows (x64 and x86) .msi and .exe, source, headers and docs (both produced currently by an OS X slave). i.e. the slaves with "arm" in their name don't need to have finished to progress to the next step. However, **if you promote builds _before_ ARM builds have finished, you must repeat the promotion step for the ARM builds when they are ready**.
 
-This sets up the branch so that nightly builds are produced with the next version number _and_ a pre-release tag.
+### 8. Tag and Sign the Release Commit
+
+Tag the release as <b><code>vx.y.z</code></b> and sign **using the same GPG key that will be used to sign SHASUMS256.txt**.
```

Comment:
We may want to state that the key may be a child/subkey of the one in the changelog.

I.e. my setup works but the release script throws throws a warning.